src/environments/environment.ts

```diff
@@ -3,7 +3,7 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  apiUrl: 'http://localhost:3000',
+  apiUrl: 'http://sensrnet-test.westeurope.cloudapp.azure.com/api',
```

Comment:
Is `env.apiUrl` mooier of `env.api.url`? Wat is gebruikelijk in Angular / NodeJs frontend apps?

Reply:
Daar is niet echt een standaard in, we kunnen het zelf noemen zoals we willen. In de Angular docs wordt ook `apiUrl` gebruikt, dus daar gaat mijn voorkeur naar uit. Ook omdat we geen andere properties gebruiken voor de api. Mochten die er later nog wel bij komen kunnen we het altijd nog nesten 😊